make staticdeps run on systems where python is 3, and 2.7 is availabl…

cyberorg opened the pull request:
<!--
 1. Following guidance below, replace …'s with your own words
 2. After saving the PR, tick of completed checklist items
 3. Skip checklist items that are not applicable or not necessary
 4. Delete instruction/comment blocks
-->

### Summary
<!--
 * description of the change
 * manual verification steps performed
 * screenshots if the PR affects the UI
-->
python is available as python2

make dist fails as it fails to run staticdeps on Ubuntu 20.04.
python build_tools/py2only.py puts just the concurrent folder in py2only, running python2 build_tools/py2only.py completes the step successfully.
…

### Reviewer guidance
<!--
 * how can a reviewer test these changes?
 * are there any risky areas that deserve extra testing
-->
On Ubuntu 20.04 or any distro where python 2.7 is available as /usr/bin/python2 
git pull the kolibri repository
Follow https://kolibri-dev.readthedocs.io/en/develop/getting_started.html to set up the build environment
Run make dist

…

### References
<!--
 * references to related issues and PRs
 * links to mockups or specs for new features
 * links to the diffs for any dependency updates, e.g. in iceqube or the perseus plugin
-->

…

----

### Contributor Checklist


PR process:

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

cyberorg committed:
make staticdeps run on systems where python is 3, and 2.7 is available as pyton2

make dist fails as it fails to run staticdeps on Ubuntu 20.04.
python build_tools/py2only.py puts just the concurrent folder in py2only, running python2 build_tools/py2only.py completes the step successfully.

rtibbles reviewed (commented):
Thanks for the contribution - please feel free to also add your name to `AUTHORS.md` as part of this PR.

As the build for this PR seems to pass fine, this looks safe to me - but just going to flag @DXCanas for additional review here.

cyberorg committed:
Update AUTHORS.md

For a very tiny contribution https://github.com/learningequality/kolibri/pull/7193

rtibbles reviewed (approved):
Approved!
